### ISSUE TITLE
Does not work with talkback on android

### ISSUE THREAD

**Issue by ArchdukeTim** [mpowaga/react-slider#263]
## Description

Cannot use talkback gestures 'swipe-up' and 'swipe-down' to change value. 

Tested on [these example](https://zillow.github.io/react-slider/)
